src/test/java/io/stargate/sgv2/jsonapi/api/v1/util/TableTemplates.java

```diff
@@ -88,17 +88,6 @@ public DataApiResponseValidator createIndex(String indexName, String column) {
     return sender.postCreateIndex(json);
   }
 
-  public DataApiResponseValidator dropIndex(String indexName) {
```

Comment:
Makes sense as this moved. But would we need "KeyspaceTemplates" or such as replacement?

Reply:
No other class using it, will add it anyways